docs/Changelog

```diff
@@ -30,10 +45,18 @@ Stable versions
 	- fix warnings in configure script
 	- fix Watcom C build on OS/2
 	- fix Amiga build
+	- several code clean-ups
 	Changes by Carsten Teibes:
 	- fix lite build mod loader symbols
 	Changes by NoSuck:
+	- add new xmp_set_row() call to skip replay to the given row
+	- IT: T00 now repeats previous slide
 	- fix IT message length
+	- prevent clobbering of muted channels' volumes in IT modules
+	- clamp number of IT envelope nodes at load time
+	- fix IT message (comment) length miscalculation
```

Comment:
Are both of these IT message length entries for the same fix or for different things?

Reply:
Yeah. Removed dups now.